web/_includes/icon.html

```diff
@@ -23,6 +23,8 @@
     <span class="fas fa-fw fa-project-diagram color-white"></span>
   {% when 'button-email' %}
     <span class="fas fa-fw fa-envelope color-white"></span>
+  {% when 'unused-openssl-icon' %}
```

Comment:
The key has a bit unintuitive name. When you look on the other icon keys, none has the word "icon" and if it's bound to a library, the library name is a prefix. I thus suggest `openssl-unused` instead.

Reply:
Furthermore, please move it higher in the file right after the library icons.